File: julia/multiples-of-3-or-5/solution.jl
# https://www.codewars.com/kata/514b92a657cdc65150000006

module Solution

  triOrQuin(n) = n % 3 == 0 || n % 5 == 0

  function sumofmultiples(n)
    sum(filter(triOrQuin, 1:n-1))
  end
  
end

File: dart/multiples-of-3-or-5/solution.dart
// https://www.codewars.com/kata/514b92a657cdc65150000006

int solution(int n) {
  int sum=0;
  for(int i=1; i<n; i++) {
    if (i % 3 == 0 || i % 5 == 0) {
      sum += i;
    }
  }
  return sum;
}

